temporian/implementation/numpy/operators/tick.py

```diff
@@ -41,14 +41,15 @@ def __call__(self, input: EventSet) -> Dict[str, EventSet]:
 
         # fill output EventSet data
         for index_key, index_data in input.data.items():
-            if len(index_data.timestamps) == 0:
+            if len(index_data.timestamps) <= 1:
```

Comment:
For brainstorming: Should there be cases where at least one event is returned?

Reply:
Changed this, the behavior now is similar to what we have currently on main.

Check the new test: https://github.com/google/temporian/pull/376/files#diff-22438ae7b5327b043eb72f32e896dc62eb5c44d523d3535a0f3b65c1339894a1R57